hosted/helpers/helper_common.go

```diff
@@ -334,14 +334,6 @@ func FilterUIUnsupportedVersions(versions []string, client *rancher.Client) (fil
 	return
 }
 
-// CheckMapElements checks if map1 keys are subset of map2
-func CheckMapKeys(map1, map2 map[string]string) (exists bool) {
```

Comment:
No harm in keeping this function

Reply:
It is not used anywhere right now, and it does not support cases where one map and empty and the other is not (for e.g. originalNGTags is non-empty, and updatedNGTags is empty, in which case it would always return false).

I think it is better to use `maps.Equal` instead of defining a similar function, unless you had a reason for defining it in the first place.

Reply:
On second thought, you might be right, this function can still be useful to check if one map is a subset of another one, I'll keep this function then. Thanks for pointing it out.